jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/skaffold/FilesMojo.java

```diff
@@ -154,7 +162,32 @@ public void execute() throws MojoExecutionException, MojoFailureException {
   }
 
   private List<Path> resolveExtraDirectories() {
-    List<File> paths = extraDirectory.getPaths();
+    // TODO: Should inform user about nonexistent directory if using custom directory.
+    String deprecatedProperty =
+        MavenProjectProperties.getProperty(PropertyNames.EXTRA_DIRECTORY_PATH, project, session);
+    String newProperty =
+        MavenProjectProperties.getProperty(PropertyNames.EXTRA_DIRECTORIES_PATHS, project, session);
+
+    List<File> deprecatedPaths = extraDirectory.getPaths();
+    List<File> newPaths = extraDirectories.getPaths();
+
+    if (deprecatedProperty != null) {
```

Comment:
You may not need the add the deprecation warning here as this is mostly an internal task?

Reply:
Ah, the output is not visible, right? I'll remove the warnings.